docs/pages/code/operations/reject.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App;
+
+use loophp\collection\Collection;
+
+include __DIR__ . '/../../../../vendor/autoload.php';
+
+$divisibleBy2 = static fn ($value): bool => 0 === $value % 2;
+$divisibleBy3 = static fn ($value): bool => 0 === $value % 3;
+
+$collection = Collection::fromIterable(range(1, 10))
+    ->reject($divisibleBy2); // [1, 3, 5, 7, 9]
+
+$collection = Collection::fromIterable(range(1, 10))
+    ->reject($divisibleBy2, $divisibleBy3); // [1, 5, 7]
+
+$collection = Collection::fromIterable(range(1, 10))
+    ->reject($divisibleBy2)
+    ->reject($divisibleBy3); // [1, 5, 7]
```

Comment:
is it just me or because of the nature of `reject`, calling it with variadic callbacks is the same as calling it multiple times? Acts as logical `AND`

Reply:
Yes exactly !

Not "A or B" is "Not A and Not B"